target/linux/armsr/armv7/config-6.1

```diff
@@ -42,36 +62,205 @@ CONFIG_CPU_SPECTRE=y
+CONFIG_I2C_MUX_PCA954x=y
+CONFIG_I2C_MUX_PINCTRL=y
+CONFIG_I2C_STM32F7=y
+CONFIG_IIO_STM32_LPTIMER_TRIGGER=y
+# CONFIG_INTEL_QEP is not set
+# CONFIG_INTERRUPT_CNT is not set
+# CONFIG_MICROCHIP_TCB_CAPTURE is not set
+CONFIG_JFFS2_FS=y
+CONFIG_MACH_STM32MP13=y
+CONFIG_MACH_STM32MP157=y
+CONFIG_MFD_CORE=y
+# CONFIG_MFD_STM32_TIMERS is not set
+# CONFIG_MTD_ST_SPI_FSM is not set
+CONFIG_MFD_STM32_LPTIMER=y
 CONFIG_MIGHT_HAVE_CACHE_L2X0=y
+CONFIG_MMC=y
+CONFIG_MMC_ARMMMCI=y
+CONFIG_MMC_CQHCI=y
+CONFIG_MMC_RICOH_MMC=y
+CONFIG_MMC_SDHCI=y
+CONFIG_MMC_SDHCI_IO_ACCESSORS=y
+CONFIG_MMC_SDHCI_PCI=y
+CONFIG_MMC_SDHCI_PLTFM=y
+CONFIG_MMC_SDHCI_ST=y
+CONFIG_MMC_STM32_SDMMC=y
 CONFIG_MODULES_USE_ELF_REL=y
+CONFIG_MTD_BLOCK=y
 CONFIG_NEON=y
 CONFIG_NR_CPUS=4
+CONFIG_NVMEM_STM32_BSEC_OPTEE_TA=y
+CONFIG_NVMEM_STM32_ROMEM=y
+CONFIG_OF_DYNAMIC=y
 CONFIG_OLD_SIGACTION=y
 CONFIG_OLD_SIGSUSPEND3=y
+CONFIG_OPTEE=y
 CONFIG_OUTER_CACHE=y
 CONFIG_OUTER_CACHE_SYNC=y
 CONFIG_PAGE_OFFSET=0xC0000000
+CONFIG_PCS_XPCS=y
 CONFIG_PERF_USE_VMALLOC=y
+CONFIG_PINCTRL_STM32=y
+CONFIG_PINCTRL_STM32MP135=y
+CONFIG_PINCTRL_STM32MP157=y
+CONFIG_PL310_ERRATA_753970=y
+CONFIG_PL310_ERRATA_769419=y
+# CONFIG_POWER_RESET_ST is not set
+CONFIG_PHYLINK=y
+CONFIG_PHY_STM32_USBPHYC=y
+# CONFIG_PHY_MIPHY28LP is not set
+CONFIG_PINCTRL_SINGLE=y
+CONFIG_POWER_RESET_GPIO=y
+CONFIG_POWER_RESET_GPIO_RESTART=y
+CONFIG_POWER_RESET_SYSCON=y
+CONFIG_POWER_RESET_SYSCON_POWEROFF=y
+CONFIG_PWM_STM32=y
+CONFIG_PWM_STM32_LP=y
+CONFIG_REGMAP_I2C=y
+CONFIG_REGMAP_SPI=y
+CONFIG_REGULATOR_GPIO=y
+CONFIG_REGULATOR_PWM=y
+CONFIG_REGULATOR_STM32_BOOSTER=m
+CONFIG_REGULATOR_STM32_PWR=y
+CONFIG_REGULATOR_STM32_VREFBUF=y
+CONFIG_RESET_CONTROLLER=y
+CONFIG_RFKILL_GPIO=y
+CONFIG_RPMSG=y
+# CONFIG_RPMSG_CHAR is not set
+# CONFIG_RPMSG_CTRL is not set
+CONFIG_RPMSG_NS=y
+CONFIG_RPMSG_TTY=y
+CONFIG_RPMSG_VIRTIO=y
+# CONFIG_RTC_DRV_ST_LPC is not set
+CONFIG_RTC_DRV_OPTEE=y
+CONFIG_RTC_DRV_STM32=y
+CONFIG_RTC_I2C_AND_SPI=y
 CONFIG_RTC_MC146818_LIB=y
-CONFIG_SERIAL_OF_PLATFORM=y
+CONFIG_SD_ADC_MODULATOR=y
+CONFIG_SENSORS_IIO_HWMON=y
+CONFIG_SERIAL_STM32=y
+CONFIG_SERIAL_STM32_CONSOLE=y
 CONFIG_SMP_ON_UP=y
+CONFIG_SND=y
+CONFIG_SND_DMAENGINE_PCM=y
+CONFIG_SND_JACK=y
+CONFIG_SND_JACK_INPUT_DEV=y
+CONFIG_SND_PCM=y
+CONFIG_SND_PCM_IEC958=y
+CONFIG_SND_SOC=y
+CONFIG_SND_SOC_GENERIC_DMAENGINE_PCM=y
+CONFIG_SND_SOC_I2C_AND_SPI=y
+CONFIG_SND_SOC_SPDIF=y
+CONFIG_SND_SOC_STM32_DFSDM=y
+CONFIG_SND_SOC_STM32_I2S=y
+CONFIG_SND_SOC_STM32_SAI=y
+CONFIG_SND_SOC_STM32_SPDIFRX=y
+# CONFIG_SND_COMPRESS_OFFLOAD is not set
+CONFIG_SND_SPI=y
+CONFIG_SOUND=y
+CONFIG_SOUND_OSS_CORE=y
+CONFIG_SPI=y
+CONFIG_SPI_DYNAMIC=y
+CONFIG_SPI_MASTER=y
+# CONFIG_SPI_ST_SSC4 is not set
+CONFIG_SPI_MEM=y
+CONFIG_SPI_SPIDEV=y
+CONFIG_SPI_STM32=y
+CONFIG_SPI_STM32_QSPI=y
+CONFIG_SRAM=y
+CONFIG_SRAM_EXEC=y
+CONFIG_STM32_EXTI=y
+CONFIG_STM32_THERMAL=y
+CONFIG_STM32_WATCHDOG=y
+CONFIG_ST_IRQCHIP=y
+# CONFIG_ST_LPC_WATCHDOG is not set
+CONFIG_ST_THERMAL=y
```

Comment:
Drop this, only STI, but why is CONFIG_STM32_THERMAL not selected? I can't find it!

Reply:
`CONFIG_STM32_THERMAL` is selected, just before `CONFIG_STM32_WATCHDOG`. Removed the rest. Thanks